### PR TITLE
removed lens and import cleanup

### PR DESCRIPTION
**Pull request by piq9117**
- removed lens and only using generic-lens. This will reduce the
dependency footprint a little bit.

**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [X] I have followed the repository's [pull request submission guidelines](../blob/master/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Redundancy in dependencies causing unnecessary dependency footprint

**Describe the solution you've provided**

Removed `lens` library. Replaced functions from `lens` with functions from `generic-lens`

**Describe alternatives you've considered**

n/a

**Additional context**

n/a

### CODE REVIEW

**Comment by louis-launchdarkly** [comment]
Hello @piq9117, thank you for your contribution! We are reviewing and testing the change, please wait and we will give an update when we finish looking at this PR.

**Comment by alexbiehl** [comment]
@louis-launchdarkly any news for us yet?

**Comment by louis-launchdarkly** [comment]
@alexbiehl, unfortunately, no update yet as there is limited Haskell expertise on the team right now. We still have this in mind and want to get through this soon (and your other contribution too).

File: src/LaunchDarkly/Server/Evaluate.hs [hunk 1]
```diff
@@ -1,17 +1,17 @@
 module LaunchDarkly.Server.Evaluate where
 
-import           Control.Lens                        ((%~))
+import           Data.Generics.Internal.VL           (over)
```
Comment (gwhelanLD) [inline src/LaunchDarkly/Server/Evaluate.hs:3]: `Data.Generics.Internal.VL` is not exposed until `generic-lens` `2.0.0.0`, and the current minimum in `package.yaml` is `1.1.0.0`.

**Pull request closed without merging**